tests/msc3030_test.go

```diff
@@ -203,6 +203,16 @@ func TestJumpToDateEndpoint(t *testing.T) {
 				contextResResBody := client.ParseJSON(t, contextRes)
```

Comment:
> Fixes https://github.com/matrix-org/synapse/issues/13944?

@squahtx I don't think this fixes that issue. The error there is:

> CSAPI.MustDoFunc response return non-2xx code: 403 Forbidden - body: {"errcode":"M_FORBIDDEN","error":"You don't have permission to access that event."}

which indicates we also need to retry the `/context` request above until it works

Reply:
Fixed in a separate PR: https://github.com/matrix-org/synapse/pull/14096